src/main/sql/schema-drop-postgres.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE  IF EXISTS BATCH_STEP_EXECUTION_CONTEXT;
```

Comment:
Isn't this and the schema file already included in the framework's JAR files?

Reply:
Maybe?  I was following the [pattern in the Spring Academy course](https://github.com/vmware-tanzu-learning/course-spring-batch-essentials-code/blob/main/src/sql/schema-drop-postgresql.sql).

Reply:
Perhaps it is in the GitHub repo for a reason that deals with the Educates platform.